app/controllers/episodes_controller.rb

```diff
@@ -26,7 +26,7 @@ def create
     @episode = Episode.new(episode_params)
 
     if @episode.save
-      redirect_to episode_path(@episode), notice: 'Episode was successfully created.'
+      redirect_to episodes_path, notice: 'Episode was successfully created.'
```

Comment:
Why are you redirecting to the index view?

Reply:
See commit message comment in 0b88a5b